cranelift/codegen/src/isa/s390x/inst.isle

```diff
@@ -1727,6 +1778,93 @@
         result))
 
 
+;; Helpers for instruction sequences ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+;; Completed instruction sequence for use in MInst.Loop.
+(type VecMInst (primitive VecMInst))
+
+;; Partial (mutable) instruction sequence in the process of being created.
+(type VecMInstBuilder extern (enum))
+
+;; Create a new empty instruction sequence builder.
+(decl iv_new () VecMInstBuilder)
```

Comment:
Does "iv" stand for "instruction vec"? For readability, what do you think of calling this `inst_builder_new` and similar for the other `iv_*` decls?

Reply:
Done.